### PR TITLE
Removed duplicated pattern

### LINKED ISSUE #3151: Duplicate filters on patterns page
#### Description

Two duplicate filters exist  on patterns page from area I section.

image

#### Expected Behavior

 remove one of them.
#### Environment:
- Host OS: 
- Browser: 

---
<h3> Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h3>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- 🎨 Wireframes and designs for Layer5 site in [Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). 
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by Nawaz027**
Signed-off-by: Nawaz027 <nawazsiddiqui27@gmail.com>

**Description**
removed one of the duplicated filters on patterns page from area I section.
Link of the webpage:
https://layer5.io/learn/service-mesh-books/service-mesh-patterns

This PR fixes https://github.com/layer5io/layer5/issues/3151

This PR fixes #3151

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit c7318899b0e588062ed693cb8482f9d0fe8c3594 at: https://63198f5e1a0850413efc040e--layer5.netlify.app

**Review by legendarykamal — CHANGES_REQUESTED** [review]
I still see both the filters 
image

File: src/collections/service-mesh-books/service-mesh-patterns/tableData.js [hunk 1]
```diff
@@ -13,11 +13,6 @@ export const Data = [
     category: "Foundational",
     subsection: "Area I"
   },
-  {
-    service_mesh_pattern: "Pattern: How a service mesh empowers a service owner",
-    category: "Foundational",
-    subsection: "Area I"
-  },
```
Comment (Nikhil-Ladha) [inline src/collections/service-mesh-books/service-mesh-patterns/tableData.js:20]: Where is the chapter repeated?

**Comment by Nawaz027** [comment]
Sorry, I accidently removed table data. Now it's being fixed. Also category column is being removed and service mesh pattern is visible now.

**Comment by legendarykamal** [comment]
@Nawaz027 please do fix the dco issue.

**Comment by leecalcote** [comment]
Creating new deployment preview...

**Comment by l5io** [comment]
🚀 Preview for commit 430d04fa7850e153017d03561603f37f585155ba at: https://632b986d2cf1860595a9f1fc--layer5.netlify.app

**Comment by Nikhil-Ladha** [comment]
I still don't get where is this text duplicated, can you please point me to it? 🤔

**Comment by legendarykamal** [comment]
hi @Nikhil-Ladha text isnt duplicated , please see the filters here https://github.com/layer5io/layer5/pull/3160#pullrequestreview-1100703670

**Comment by Nikhil-Ladha** [comment]
Ok, I see. Perhaps a better issue description is needed...

**Comment by legendarykamal** [comment]
> Ok, I see. Perhaps a better issue description is needed...

updated

**Comment by debo19** [comment]
@Nawaz027 will you discuss this on the websites call today?

**Comment by Nikhil-Ladha** [comment]
@Nawaz027 any updates here?

**Comment by leecalcote** [comment]
Other than DCO, is there do you approve, @legendarykamal?

**Comment by leecalcote** [comment]
Thanks for jumping and helping, @Nawaz027. Much appreciated.

There does not seem to be a duplication of patterns, which leaves me wondering what is being fixed here. It seems that we are deleting a valid pattern in this PR.

**Comment by leecalcote** [comment]
@Nawaz027 or @legendarykamal will you take a moment to confirm, please?

**Review by legendarykamal — APPROVED** [review]
other than DCO, everything looks good @leecalcote .

**Review by legendarykamal — COMMENTED** [review]
Let us close this as it has been already successfully solved in another PR by @tiger-yash .

Thank you @Nawaz027 for contributing .

**Pull request closed without merging**